packages/rancher-istio/charts/configs/istio-base.yaml

```diff
@@ -2,29 +2,38 @@ apiVersion: install.istio.io/v1alpha1
 kind: IstioOperator
 spec:
   addonComponents:
-    grafana:
-      enabled: {{ .Values.grafana.enabled }}
     istiocoredns:
       enabled: {{ .Values.istiocoredns.enabled }}
-    kiali:
-      enabled: {{ .Values.kiali.enabled }}
-    prometheus:
-      enabled: {{ .Values.prometheus.enabled }}
-    tracing:
-      enabled: {{ .Values.tracing.enabled }}
   components:
     base:
       enabled: {{ .Values.base.enabled }}
-    citadel:
-      enabled: {{ .Values.citadel.enabled }}
     cni:
       enabled: {{ .Values.cni.enabled }}
     egressGateways:
     - enabled: {{ .Values.egressGateways.enabled }}
-      name: istio-egressgateway
     ingressGateways:
     - enabled: {{ .Values.ingressGateways.enabled }}
-      name: istio-ingressgateway
+      k8s:
```

Comment:
believe this section should be along with `name: istio-ingressgateway`, as in example: https://istio.io/latest/docs/setup/install/istioctl/#configure-gateways